runtime/onert/core/include/ir/OperandIndexSequence.h

```diff
@@ -27,6 +27,12 @@ namespace onert
 namespace ir
 {
 
+enum class Remove
+{
+  duplicated = 0x1,
+  optional = 0x10
```

Comment:
I think we use all uppercase letters for enum names.

Reply:
@wateret I didn't know the rule. I had not seen open source that uses all CAPITAL for enum name in recent years. Anyway, I've found `onert` (that is, under `onert`) follow the rule though other sources out of `onert` in this repo do not follow the rule. I will update.